water_neuron/src/conversion.rs

```diff
@@ -79,6 +77,7 @@ pub async fn nicp_to_icp(arg: ConversionArg) -> Result<WithdrawalSuccess, Conver
                 Ok(WithdrawalSuccess {
                     withdrawal_id,
                     block_index,
+                    icp_amount: Some(icp_due),
```

Comment:
why are we now tracking the `icp_amount`? Are we doing something with it in the front-end?

Reply:
It's to improve interactions with canisters.